react-native-libraries.json

```
@@ -6868,10 +6868,16 @@
   {
     "githubUrl": "https://github.com/mateusz1913/react-native-avoid-softinput",
```

```suggestion
    "githubUrl": "https://github.com/mateusz1913/react-native-avoid-softinput/tree/master/packages/react-native-avoid-softinput",
```

If you change the GitHub URL to directly point to the package, not the whole monorepo, the automatic detection should work. This is also the correct way to specify packages inside one, using the repo URL in that case can also affect metadata, not only new architecture detection.